tests/sqlparser_postgres.rs

```diff
@@ -401,6 +401,109 @@ PHP	₱ USD $
     //assert_eq!(sql, ast.to_string());
 }
 
+#[test]
+fn parse_update_set_from() {
+    let sql = "UPDATE t1
+    SET name = t2.name FROM ( \
+        SELECT name, id \
+        FROM t1 \
+        GROUP BY id \
+    ) t2 \
+    WHERE t1.id = t2.id";
+    let ast = pg().one_statement_parses_to(sql, "");
```

Comment:
Would it be possible to follow the model of the test `test_copy_from` (which is immediately below this one)?

So something like

```rust
    let stmt = pg().verified_stmt(sql);
    assert_eq!(
        stmt,
        Statement::Update {
...
    });
```

Reply:
Good suggestion, I didn't notice that before.

Changed.